custom_components/tuya_local/devices/eurom_sani_heater.yaml

```diff
@@ -0,0 +1,103 @@
+name: Heater
+products:
+  - id: vx9tbbzhfmcc0f3k
+    manufacturer: Eurom
+    model: Sani 800
+entities:
+  - entity: climate
+    translation_only_key: heater
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: "off"
+          - dps_val: true
+            value: heat
+      - id: 2
+        type: integer
+        name: temperature
+        range:
+          min: 0
+          max: 37
+        mapping:
+          - constraint: temperature_unit
+            conditions:
+              - dps_val: true
+                value_redirect: temp_set_f
+                range:
+                  min: 32
+                  max: 99
+      - id: 3
+        type: integer
+        name: temp_set_f
+        optional: true
+        hidden: true
+        range:
+          min: 32
+          max: 99
+      - id: 4
+        type: boolean
+        name: temperature_unit
+        optional: true
+        mapping:
+          - dps_val: false
+            value: C
+          - dps_val: true
+            value: F
+      - id: 5
+        type: integer
+        name: current_temperature
+        mapping:
+          - constraint: temperature_unit
+            conditions:
+              - dps_val: true
+                value_redirect: temp_current_f
+      - id: 6
+        type: integer
+        name: temp_current_f
+        optional: true
+        hidden: true
+  - entity: time
+    name: Countdown time
+    category: config
+    dps:
+      - id: 7
+        type: integer
+        name: minute
+        range:
+          min: 0
+          max: 1440
+        mapping:
+          - step: 60
+      - id: 8
+        type: boolean
+        name: available
+  - entity: switch
+    name: Countdown
+    icon: "mdi:timer"
+    category: config
+    dps:
+      - id: 8
+        type: boolean
+        name: switch
+        optional: true
+  - entity: select
+    name: Schedule
+    icon: "mdi:calendar-clock"
+    category: config
+    dps:
+      - id: 9
+        type: string
+        name: schedule_data
```

Comment:
Should be a text entity, hidden by default, to make it available to advanced users.